Makefile

```diff
@@ -0,0 +1,4 @@
+# This file is used for aliases for long commands such as clang-format
+files := $(shell find . \( -name '*.h' -or -name '*.hpp' -or -name '*.cpp' \) -not -path "./external/*")
```

Comment:
this script ignores files in the ./external folder which we don't have. Change to third_party or remove